src/components/challenges/ChallengesList.js

```diff
@@ -1,36 +1,45 @@
-import React from 'react'
+import React from "react";
 
 // custom components
-import useFirestore from '../../hooks/useFirestore';
-import Hero from '../dashboard/Hero';
-import ChallengeCard from './ChallengeCard'
-import challengeLottie from '../../assets/animated_illustrations/challenge.json'
-import SkeletonCard from '../skeletons/SkeletonCard';
```

Comment:
please don't add make this type of commit(single quote to double quote), it would be very difficult for me to see the actual changes.

Reply:
you need to add prettier.rc file to project to avoid this